privx_api/privx_api.py

```diff
@@ -386,6 +402,27 @@ def create_user(self, user: dict) -> PrivXAPIResponse:
         response = self._http_post("userstore.users", body=user)
         return PrivXAPIResponse(response, 201)
 
+    def get_local_users(self) -> PrivXAPIResponse:
+        """
+        Get users.
+
+        Returns:
+            PrivXAPIResponse
+        """
+        response = self._http_get("userstore.users")
+        return PrivXAPIResponse(response, 200)
+
+    def delete_user(self, user_id: str) -> PrivXAPIResponse:
+        """
```

Comment:
for consistency this should be renamed as `delete_local_user()`

Reply:
renamed couple of functions to create_local_user() and delete_local_user()